### DISCUSSION TITLE
Max Distance Integer

### DISCUSSION THREAD

**Discussion by jmcclure0921** [ESPresense/ESPresense#56] [Q&A] [answered]
Is max Distance an integer due to technical limitations or for simplicity?  I love the project, but one of my rooms reaches a little too far a 3 and a little too short at 2.

**Comment by DTTerastar** [comment]
A library I used didn't have a float setting, but I was able to add one

**Comment by DTTerastar** [comment, marked as answer]
1.6.2 is out

**Reply by jmcclure0921** [reply]
You are the man!
